app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -1,19 +1,16 @@
 .gem-c-feedback {
   background: govuk-colour("white");
-  margin: 0 auto;
+  margin: govuk-spacing(6) auto 0 auto;
   max-width: $govuk-page-width;
   position: relative;
 
-  .visually-hidden {
-    @include govuk-visually-hidden;
-  }
-}
-
-.gem-c-feedback--top-margin {
-  margin-top: govuk-spacing(6);
-
   @include govuk-media-query($from: tablet) {
     margin-top: govuk-spacing(9);
+    border-bottom: 1px solid govuk-colour("white");
```

Comment:
Suggestion: currently this swaps between the `.gem-c-feedback` element having the border bottom above tablet and the `.gem-c-feedback__prompt-questions` having it on mobile (where both need it to have two, when stacked) - would it be possible to just have the border bottom on the `.gem-c-feedback__prompt-questions` elements and not need any media queries involved?

Reply:
Actually, I've just realised why this is difficult - at a certain breakpoint the heights of the two elements are inconsistent. Hmm. Maybe don't worry about this. Although maybe there's a solution..

![Screenshot 2020-03-11 at 13 47 42](https://user-images.githubusercontent.com/861310/76423630-ff740e80-639e-11ea-9807-bfdedd6b9f2e.png)

Reply:
Applying the border to `.gem-c-feedback__prompt-questions` was my first approach, but I also noticed the issue you had above so switched approaches. I tried again, but I think either way will require media queries.

Reply:
Yeah, fair enough.